File: Week01-C/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int get_height(void);  // Function to get user input
void print_tower(int); // Function to print the tower

int main(void)
{
    // Get height from user
    int n = get_height();

    // Print result
    print_tower(n);
}

int get_height(void)
{
    int n;
    do
    {
        n = get_int("Enter height of the tower: ");
    } while (n < 1 || n > 8);
    return n;
}

void print_tower(int n)
{
    for (int i = 1; i <= n; i++)
    {

        // For spaces left of the tower
        for (int k = n - i; k > 0; k--)
        {
            printf(" ");
        }

        // Left tower
        for (int j = 0; j < i; j++)
        {
            printf("#");
        }

        printf("  ");

        // Right tower
        for (int m = 0; m < i; m++)
        {
            printf("#");
        }

        printf("\n");
    }
}

File: Week06-Python/problemset/sentimental-mario-more/mario.py
# TODO
from cs50 import get_int


def main():
    # get height from user
    height = get_height()

    # Print tower
    print_tower(height)


def get_height():
    while True:
        n = get_int("height: ")
        if 1 <= n <= 8:
            return n


def print_tower(height):
    for row in range(1, height + 1):
        print(f'{" " * (height - row)}{row * "#"}  {row * "#"}')


if __name__ == "__main__":
    main()
